### PR TITLE
Added Community Social Media Badge

### PR DESCRIPTION
**Pull request by dth99**
Added Community Social Media Badge in README.md

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by dth99** [comment]
> LGTM rocket Thank you for your contribution! smile+1

thank you sir

**Comment by Panquesito7** [comment]
@eddiejaoude, can you please review this, please? Seems like nobody else has access rather than Allan Regush and Sara.

**Review by Evavic44 — APPROVED** [review]
LGTM! 👍🏾

File: README.md [hunk 1]
```diff
@@ -14,4 +14,6 @@ Website built using [Kartra](http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp) a low c
 
 <a href="http://discord.eddiehub.org"><img src="https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white"><br>
 <a href="https://github.com/EddieHubCommunity"><img src="https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white" width="102px"></a><br>
-<a href="https://www.youtube.com/eddiejaoude"><img alt="Surpath YouTube" src="https://img.shields.io/badge/YouTube-FF0000?style=for-the-badge&logo=youtube&logoColor=white"></a>
+<a href="https://www.youtube.com/eddiejaoude"><img alt="Surpath YouTube" src="https://img.shields.io/badge/YouTube-FF0000?style=for-the-badge&logo=youtube&logoColor=white"></a><br>
```
Comment (eddiejaoude) [inline README.md:17]: Surpath?

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Comment by dth99** [comment]
> Thank you +1

thank you sir

**Pull request merged**